tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,155 @@
+
+#### Tasking Assignment
+
+Contributors should assign themselves to various outline areas. This allows focus and identifies gaps needing more contributors. Use tools like Google Docs' "assign to me" feature for task assignment. Alternatively, participants may tag themselves in line by typing their name, or other agreed upon option to clarify who is responsible for which portions of the paper.
+
+#### Content Rough-in
+
+Assignees provide rough content through phrases, paragraphs, or bullets. If the assignee has drafted complete content (full sentences, ideas, well-expressed) they should indicate the content as "draft". This informs other collaborators that less effort may be needed to further develop the content and may facillitate comments to refine the material further or restructure as content is further developed.
+
+#### Collaborative Review
+
+Contributors refine raw content into drafts, comment on sections, expand ideas, and provide citations. It is recommended contributors use the corresponding Slack channel for larger thematic discussions. Groups may wish to establish a separate channel for technical paper discussions, but are not required to. The Issue should provide the link for the primary commnication channel should anyone have questions.
+
+#### Executive Summary and Content Wrap-up
+
+If the paper exceeds five pages, write an executive summary no more than three paragraphs. Executive summaries are intended to be somewhat generalized and provide an overview of the paper without diving into the details. It allows readers to ascertain relevance of the content before committing to read it in entirety to ascertain value. Once complete, finalize content and discussions internal to the group.
+
+#### Narrative Voice
+
+Assign no more than three individuals for this task to ensure a unified voice. Ensure consistent language, phrasing, acronym use, footnotes, and citations. The final result of this unification should ensure the paper reads consistently, like it was written by a single entity. This also functions as the final edit before group review.
+
+#### Final Group Review
+
+The group collectively reviews the polished paper before opening it for community review. This is the last cursory check before opening to public content. It is imperative that contributors verify all referencable items are linked, cited, concepts explained, acronyms spelled out, and any unclear content is corrected. If the group fails to gain consensus on any of the content presented they may choose to remove it or explain the different perspectives that allow the reader to draw their own conclusions, this is paritcularly important where a technical recommendation results in security or performance trade offs.
+
+The group should assign individuals to serve as comment adjudicators for public review at this point. These individuals will be responsible for validating commentary against the defined scope, assumptions, and audience.
```

Comment:
Are there any requirements regarding the adjudicators? E.g. they may not have been authors of the whitepaper, or the whitepaper lead etc?

Reply:
added below in the adjudicators area.